src/main/java/uk/gov/register/util/enums/DataType.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.register.util.enums;
+
+public enum DataType {
```

Comment:
We have an existing `Datatype` interface which may get confusing with this `enum`. Given that we're storing the pattern separately from this enum, it might be worth doing a comparison of the field's name against `"datetime"` or its type against `DatetimeDatatype` in the ItemValidator instead?

Reply:
Yes, I think something like `isinstanceof` `DatatimeDatatype` would make sense